### PR TITLE
Add kustomization configuration for a px cli demo

### PR DESCRIPTION
**Pull request by ddelnano**
This adds the kustomization config needed to install a px cli demo for a cassandra application. After this is merged, I will follow up with the pixie repo change which will document how to use the kustomized files from this repo and add in the other dependencies (k8ssandra, etc).

## Testing done
- [x] Verified that `cd kubernetes; kompose convert -f ../docker-compose.yml` results in a noop ([P397](https://phab.corp.pixielabs.ai/P397))
- [x] Deployed this to a pixie cluster and verified that the `http://petclinic-backend:9966/petclinic/api/pets` endpoint and cassandra database receive constant traffic

Screenshot 2023-05-25 at 2 37 58 PM
Screenshot 2023-05-25 at 2 38 14 PM

### CODE REVIEW

File: Dockerfile-frontend [hunk 1]
```diff
@@ -27,7 +30,8 @@ RUN echo "export const environment = {production: false, REST_API_URL:  'http://
 # Use the DIST folder to package an image with NGINX.
 ####################################################################
 
-FROM nginx:1.19.4 AS runtime
+FROM $DOCKER_HUB/library/nginx:$NGINX_VERSION AS runtime
+
```
Comment (ddelnano) [inline Dockerfile-frontend:34]: I made this change upstream as well https://github.com/spring-petclinic/spring-petclinic-reactive/pull/15

File: docker-compose.yaml [hunk 2]
```diff
@@ -25,6 +26,7 @@ services:
       - 9966:9966
       - 9967:9967
     environment:
+      CASSANDRA_USE_ASTRA: "false"
```
Comment (ddelnano) [inline docker-compose.yaml:29]: I tried to keep all of our edits in the `kustomization.yaml`, however, I found it useful to run the local docker-compose setup. This was the minimum change needed to make docker-compose work without the cloud db they advertise.

File: src/main/java/org/springframework/samples/petclinic/visit/VisitReactiveServices.java [hunk 3]
```diff
@@ -91,4 +98,20 @@ public Mono<Void> deleteVisitById(UUID visitId) {
...
+     * Here we enter default values for Pets and Pet visits.
+     */
+    @Override
+    public void afterPropertiesSet() throws Exception {
+        UUID petUuid = UUID.fromString("11111111-1111-1111-1111-111111111111");
+        Mono.from(petDao.upsert(new PetEntity(UUID.randomUUID(), petUuid,
+                "Dog", "Ollie", LocalDate.of(2021, 5, 10)))).subscribe();
+        for (int i = 0; i < 10; i++)
+            Mono.from(visitDao.upsert(new VisitEntity(petUuid, UUID.randomUUID(),
+                    "General check up for Ollie", LocalDate.of(2023, 1, 25)))).subscribe();
```
Comment (ddelnano) [inline src/main/java/org/springframework/samples/petclinic/visit/VisitReactiveServices.java:115]: This ensures that the load generator container receives data when it makes requests to `http://petclinic-backend:9966/petclinic/api/pets`.

**Comment by ddelnano** [comment]
@aimichelle this is ready for review.

**Review by aimichelle — APPROVED** [review]

**Pull request merged**